packages/graphql-auth-transformer/src/resources.ts

```diff
@@ -1063,4 +1063,40 @@ identityClaim: "${rule.identityField || rule.identityClaim || DEFAULT_IDENTITY_F
             },
         });
     }
+
+    /**
+     * ES EXPRESSIONS
+     */
+
+    public makeESItemsExpression() {
+        // generate es expresion to appsync
+        return compoundExpression([
+            set(ref('es_items'), list([])),
+            forEach(
+                ref('entry'),
+                ref('context.result.hits.hits'),
```

Comment:
Is context.results.hits.hits intentional?

Reply:
Yup that's the response given back from elasticsearch
A hits object which contains list of hits.